servers/audio/effects/audio_effect_limiter.cpp

```diff
@@ -29,38 +29,66 @@
 /*************************************************************************/
 
 #include "audio_effect_limiter.h"
+#include "core/math/math_funcs.h"
 
+/*Soft clipping starts to reduce the peaks a little below the threshold level
+ *and progressively increases its effect as the input level increases such that
+ *the threshold is never exceeded.
+*/
+
+//I'm implementing this as a logistics function because it's designed to increase near
+//linearly and taper up to a certain max value, which is exactly what we want for
+//this soft clipping function
 void AudioEffectLimiterInstance::process(const AudioFrame *p_src_frames, AudioFrame *p_dst_frames, int p_frame_count) {
-	float threshdb = base->threshold;
+	//threshold_db: Threshold from which the limiter begins to be active, in decibels. Value can range from -30 to 0.
+	float threshold = Math::db2linear(base->threshold);
+
+	//ceiling: The waveform's maximum allowed value, in decibels. Value can range from -20 to -0.1.
 	float ceiling = Math::db2linear(base->ceiling);
-	float ceildb = base->ceiling;
-	float makeup = Math::db2linear(ceildb - threshdb);
-	float sc = -base->soft_clip;
-	float scv = Math::db2linear(sc);
-	float peakdb = ceildb + 25;
-	float scmult = Math::abs((ceildb - sc) / (peakdb - sc));
+
+	//soft_clip: Applies a gain to the limited waves, in decibels. Value can range from 0 to 6.
+	//This will be the slope (k) of the logistics curve. 0 will end up creating a hard clip
```

Comment:
```suggestion
	// Applies a gain to the limited waves, in decibels. Value can range from 0 to 6.
	// This will be the slope (k) of the logistics curve. 0 will end up creating a hard clip.
```